jenkins/Jenkinsfile

```diff
@@ -203,6 +209,105 @@ EOT
+                    Write-Host "Downloading Erlang..."
+                    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+                    Invoke-RestMethod -uri "${env.ERLANG_DOWNLOAD_URL}" -outfile "${env.ERLANG_INSTALLER_PATH}"
+                  } else {
+                    Write-Host "Found" ${env.ERLANG_INSTALLER_PATH} "in cache. Not downloading Erlang."
+                  }
+                  Write-Host "Installing Erlang..."
+                  Start-Process "${env.ERLANG_INSTALLER_PATH}" -ArgumentList /S -NoNewWindow -Wait
+                """
+                powershell """
+                  if (-Not (Test-Path "${env.RABBITMQ_INSTALLER_PATH}")) {
+                    Write-Host "Downloading Rabbit..."
+                    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+                    Invoke-RestMethod -uri "${env.RABBITMQ_DOWNLOAD_URL}" -outfile "${env.RABBITMQ_INSTALLER_PATH}"
+                  } else {
+                    Write-Host "Found" ${env.RABBITMQ_INSTALLER_PATH} "in cache. Not downloading RabbitMQ."
+                  }
+                  Write-Host "Installing RabbitMQ..."
+                  # Don't wait for rabbitmq installer as it seems to never return when run silently
+                  Start-Process "${env.RABBITMQ_INSTALLER_PATH}" -ArgumentList /S,/QN -NoNewWindow
+                  for (\$attempt=1; \$attempt -le 60; \$attempt=\$attempt+1) {
+                    \$status=(Get-Service RabbitMQ).Status
+                    if ("\$status" -eq "Running") {
+                      Write-Host "RabbitMQ is running."
+                      Break
+                    } else {
+                      if ("\$status" -eq "Stopped") {
+                        Write-Host "Saw RabbitMQ stopped, attempting to start."
+                        Start-Service RabbitMQ
+                      }
+                      Write-Host "Waiting for RabbitMQ to start (Attempt: \$attempt, State: \$status)..."
+                      Sleep 2
+                    }
+                  }
+                  if ("\$status" -ne "Running") {
+                    Write-Host "WARNING: RabbitMQ DID NOT START"
+                    \$host.SetShouldExit(1)
+                  }
+                """
+                echo 'Install python'
+                powershell """
+                  echo "### Install Chocolatey ###"
+                  Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+                  choco install -y python --version=3.6.0
+                  choco install -y git
+                """
+                echo 'Configure python'
+                powershell """
+                  SET \$PATH=${env.PYTHON};\$PATH
```

Comment:
If we do the install using either the embedded approach we use for the agent itself or using a venv then we don't need to mess around with paths like this.

Reply:
@yuvlar since you update the windows pipelines any ideas why you did this ?

Reply:
In fact, I see we do use a venv later... so this seems like it should be unnecessary?

Reply:
again, pasted it from AppVeyor, which was working fine.
It passed a review by Igor and it passes in the CI.
If there's an issue with it let's open a Jira and I'll take care of it